src/pages/index.tsx

```diff
@@ -1,38 +1,58 @@
 import styles from "./index.module.scss";
 import { Layout } from "src/components/global/layout.module";
-import { Game } from "@structs/game";
-import { prisma } from "src/server/prisma";
-import { trpc } from "@utils/trpc";
-import GameComponent from "@components/index/game.module";
+import { TextInput } from "@components/input/text_input.module";
+import { Button } from "@components/input/button.module";
+import { useState } from "react";
+import { Link } from "@components/input/link.module";
 
-const Home = ({ games }: Props) => {
-  const test = trpc.game.all.useQuery({});
-  if (test.data) console.log({ test });
-  console.log({ games });
+const Home = () => {
+  const [joinCode, setJoinCode] = useState("");
+
+  const fetchSession = async () => {
+    // TODO make this do something
+  };
 
   return (
-    <Layout>
+    <Layout title="Aracardi">
       <main className={styles.main}>
-        {games && games.map((game) =>
-          <GameComponent
-            key={game.id}
-            game={game}
-          />
-        )}
+
+        <h1 className={styles.title}>Play now!</h1>
+        <hr />
```

Comment:
We may want to do this as a `div` styled with CSS instead:

> Historically, this has been presented as a horizontal rule or line. While it may still be displayed as a horizontal rule in visual browsers, this element is now defined in semantic terms, rather than presentational terms, so if you wish to draw a horizontal line, you should do so using appropriate CSS.

https://developer.mozilla.org/en-US/docs/Web/HTML/Element/hr